bundles/best_practices.rst

```diff
@@ -68,30 +68,41 @@ The basic directory structure of an AcmeBlogBundle must read as follows:
 .. code-block:: text
 
     <your-bundle>/
-    ├─ AcmeBlogBundle.php
-    ├─ Controller/
-    ├─ README.md
-    ├─ LICENSE
-    ├─ Resources/
-    │   ├─ config/
-    │   ├─ doc/
-    │   │  └─ index.rst
-    │   ├─ translations/
-    │   ├─ views/
-    │   └─ public/
-    └─ Tests/
+    ├── config/
+    ├── docs/
+    │   └─ index.rst
+    ├── public/
+    ├── src/
+    │   ├── Controller/
+    │   ├── DependencyInjection/
+    │   └── AcmeBlogBundle.php
+    ├── templates/
+    ├── tests/
+    ├── translations/
+    ├── LICENSE
+    └── README.md
+
+and the bundle path must be adjusted to the root directory::
+
+    class AcmeBlogBundle extends Bundle
+    {
+        public function getPath(): string
```

Comment:
BTW, should we deprecate not overriding this method, so that we can make this the default in sf 6.0?

Reply:
I was wondering the same thing, I think we can do that when we decide to start depreciating the old structure.